### PR TITLE
Clean up BuildFiles under MuonAnalysis/

### PR DESCRIPTION
**Pull request by fwyzard**
Clean up BuildFile.xml filea with `xmllint --format`.

### CODE REVIEW

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by fwyzard** [comment]
@cmsbuild, please test

File: MuonAnalysis/MomentumScaleCalibration/bin/BuildFile.xml [hunk 1]
```diff
@@ -1,18 +1,27 @@
-<use   name="DataFormats/FWLite"/>
-<use   name="FWCore/FWLite"/>
-<use   name="boost"/>
-<use   name="PhysicsTools/FWLite"/>
+<use name="boost"/>
```
Comment (silviodonato) [inline MuonAnalysis/MomentumScaleCalibration/bin/BuildFile.xml:1]: @fwyzard

Comment (fwyzard) [reply MuonAnalysis/MomentumScaleCalibration/bin/BuildFile.xml:1]: sorted

**Comment by silviodonato** [comment]
merge

**Comment by santocch** [comment]
+1

**Pull request merged**